### PR TITLE
update exercises format wording

### PR DESCRIPTION
**Pull request by zanderhavgaard**
- update formatting and some wording of exercises
- rename exercise 10 to 08, remove numbering from sharing-images exercise
- udpate multi-container app to make it less confusing, by using DNS name instead of internal IPs

### CODE REVIEW

File: labs/03-deletion.md [hunk 1]
```diff
@@ -9,31 +9,42 @@ Spin up the container with `docker run -ti alpine`
...
+rm -rf /
+```
+
+Try to navigate around to see how much of the OS is gone: run the `ls`, `whami` and `date`.
+They should all echo back that the binary is not found.
+
+Exit out by pressing `Ctrl+d` and create a new instance of the Alpine image and look a bit around:
 
 ```
-# ls
```
Comment (sofusalbertsen) [inline labs/03-deletion.md:24]: Why are we deleting this part?

Comment (sofusalbertsen) [reply labs/03-deletion.md:24]: and the 5 lines below as well.

File: labs/06-volumes.md [hunk 2]
```diff
@@ -113,10 +118,27 @@ Multiple containers can attach to the same volume with data. Docker doesn't hand
 
 Let's try to go in and make a new html page for nginx to serve. We do this by making a new ubuntu container that has the `data` volume attached to `/tmp`, and thereafter create a new html file with the `echo` command:
 
-```bash
-docker run -ti --rm -v data:/tmp ubuntu bash
-root@9c36fcfcc048:# echo "<html><h1>hello world</h1></html>" > /tmp/hello.html
-root@9c36fcfcc048:# ls /tmp
+Start the container:
+
+```
```
Comment (sofusalbertsen) [inline labs/06-volumes.md:123]: bash in the code snippet format?

File: labs/03-deletion.md (hunk 1 above)
Comment (zanderhavgaard) [reply labs/03-deletion.md:24]: In an attempt to not make the "shell snippets" confusing, and to be consistent with the other shell snippet formats. 

The commands to run are listed above.

Comment (sofusalbertsen) [reply labs/03-deletion.md:24]: Ahh sure, did not see the listing of the commands :+1:

File: labs/06-volumes.md (hunk 2 above)
Comment (zanderhavgaard) [reply labs/06-volumes.md:123]: Sure, but would rather be consistent, the base annotation can create some weird syntax highlighting when mixing docker run commands and actual shell commands after that :shrug:

**Pull request merged**